setup.cfg

```diff
@@ -21,3 +22,12 @@ python_files =
     test_*.py
     *_test.py
     tests.py
+
+[mypy]
+python_version = 3.7
+warn_unused_configs = True
+warn_redundant_casts = True
+warn_unused_ignores = True
+
+[mypy-six.*,tornado.*,opentracing.*,thrift.*,threadloop.*,prometheus_client.*]
+ignore_missing_imports = True
```

Comment:
What does this do?

Reply:
So, by default mypy complains about imports to other libraries which have no types/type stubs. With that, I just tell mypy, that it should ignore it.

Reply:
I agree. I guess there is already a open issue there